### DISCUSSION TITLE
Casi d'uso Trascr_014, Trascr_020, Trascr_021: obbligatorietà genitori

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#940] [Supporto servizi cooperativi] [answered]
Per il caso d'uso in oggetto (_Trascrizione di nascita: Trascrizione della sentenza di adozione di minore in casi particolari emessa dal Tribunale dei minori_) risultano obbligatori sia il padre che la madre. All' art. 44, comma c) della legge 183/1984 si prevede però anche la possibilità che l'adozione sia pronunciata in favore di cittadini non coniugati; si rende pertanto necessario rivedere le obbligatorietà attualmente previste in maniera tale che sia possibile indicare anche solo il padre ovvero solo la madre.

La questione si pone in termini del tutto analoghi anche per i casi d'uso Trascr_020, Trascr_021

**Comment by mttfranci** [comment, marked as answer]
Renderemo obbligatorio l'inserimento di un solo genitore e non di entrambi.

Saluti 

Assistenza ANSC

**Comment by antonioluberto** [comment]
Grazie.
Importante è che si possano gestire distintamente i tre casi:
1) adozione pronunciata in favore di soggetto non coniugato di sesso maschile (padre)
2) adozione pronunciata in favore di soggetto non coniugato di sesso femminile (madre)
3) adozione pronunciata in favore di coppia di soggetti (padre e madre) coniugati

**Reply by mttfranci** [reply]
Una volta rilassata l'obbligatorietà sul fatto che padre e madre siano entrambi presenti queste tre casistiche saranno consentite. Da notare che il sistema ANSC non effettua però nessuna verifica sul fatto che i soggetti siano coniugati o meno.

Saluti

Assistenza ANSC

**Comment by asqueglia** [comment]
Buongiorno la segnalazione è stata corretta con il rilascio odierno della versione 1.30.0
